python/ribasim/tests/test_model.py

```diff
@@ -203,15 +203,39 @@ def test_xugrid(model, tmp_path):
     assert uds.attrs["Conventions"] == "CF-1.9 UGRID-1.0"
 
     with pytest.raises(FileNotFoundError, match="Model must be written to disk"):
-        model.to_xugrid(add_flow=True)
+        model.to_xugrid(flow=True)
 
     model.write(tmp_path / "ribasim.toml")
-    with pytest.raises(FileNotFoundError, match="Cannot find results"):
-        model.to_xugrid(add_flow=True)
-    with pytest.raises(FileNotFoundError, match="or allocation is not used"):
-        model.to_xugrid(add_flow=False, add_allocation=True)
+    with pytest.raises(FileNotFoundError, match="Cannot find basin.arrow"):
+        model.to_xugrid(flow=True)
+    with pytest.raises(FileNotFoundError, match="Cannot find basin.arrow"):
+        model.to_xugrid(flow=False, allocation=True)
     with pytest.raises(ValueError, match="Cannot add both allocation and flow results"):
-        model.to_xugrid(add_flow=True, add_allocation=True)
+        model.to_xugrid(flow=True, allocation=True)
+
+
+@pytest.mark.parametrize(
+    "model",
+    [basic_model(), outlet_model(), pid_control_equation_model(), trivial_model()],
+)
+def test_to_fews(model, tmp_path):
+    region_home = tmp_path
+    network_dir = region_home / "Config/MapLayerFiles/{ModelId}"
+    network_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
This should only create region_home if it not exists, because the code above doesn't test for the existence of the folders below it.

Reply:
I don't understand your comment. Are you saying that `exist_ok=True` is not needed. That may be true usually since tmp_path is generally not reused. This felt a bit more robust for debugging.

Reply:
Sorry, I meant that the normal code doesn't check whether the folders exist, it just tries to write. I'm not sure if pandas creates intermediate folders.

The fact that you create all intermediate folders for a test implies that it doesn't, and that's something we should catch/fix?

Reply:
Oh yeah I didn't make it create all paths on purpose since if it wouldn't exist it probably wouldn't be a correct destination.

Though we do create the whole path on model.write, and it may be convenient to be able to write to a region_home that doesn't exist in case you want to inspect the output or need to send it to somewhere. So I'll change that and remove this.